Description: Documenting maj_flt and min_flt output specifiers in ps.1
Bug-Debian: http://bugs.debian.org/434221
Author: Craig Small <csmall@debian.org>
Last-Update: 2010-02-25
--- a/ps/ps.1
+++ b/ps/ps.1
@@ -4,7 +4,7 @@
 .\" Quick hack conversion by Albert Cahalan, 1998.
 .\" Licensed under version 2 of the Gnu General Public License.
 .\"
-.TH PS 1 "July 28, 2004" "Linux" "Linux User's Manual"
+.TH PS 1 "February 25, 2010" "Linux" "Linux User's Manual"
 .\"
 .\" To render this page:
 .\"    groff -t -b -man -X -P-resolution -P100 -Tps ps.1 &
@@ -1084,6 +1084,14 @@
 (alias\ \fBspid\fR,\ \fBtid\fR).
 T}
 
+maj_flt	MAJFLT	T{
+The number of major page faults that have occured with this process.
+T}
+
+min_flt	MINFLT	T{
+The number of minor page faults that have occured with this process.
+T}
+
 ni	NI	T{
 nice value. This ranges from 19 (nicest) to \-20 (not\ nice to\ others),
 see\ \fInice\fR(1).  (alias\ \fBnice\fR).
